src/main/java/org/schabi/newpipe/extractor/utils/ExtractorHelper.java

```diff
@@ -0,0 +1,53 @@
+package org.schabi.newpipe.extractor.utils;
+
+import org.schabi.newpipe.extractor.Info;
+import org.schabi.newpipe.extractor.InfoItem;
+import org.schabi.newpipe.extractor.InfoItemCollector;
+import org.schabi.newpipe.extractor.ListExtractor;
+import org.schabi.newpipe.extractor.stream.StreamExtractor;
+import org.schabi.newpipe.extractor.stream.StreamInfo;
+import org.schabi.newpipe.extractor.stream.StreamInfoItemCollector;
+
+import java.util.Collections;
+import java.util.List;
+
+public class ExtractorHelper {
```

Comment:
What is this class for? Can't this be done by the collectors?

Reply:
I've created this class to remove some duplicated code. But you're right this may be done in the collectors. I'll have a look at it.

Reply:
All right thank you :)